packages/dduf/README.md

```diff
@@ -0,0 +1,36 @@
+# 🤗 Hugging Face DDUF
+
+Utilities to convert a string or URL to a [Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob) object, whether it represents a local file or remote URL.
```

Comment:
```suggestion
Utilities to convert a string or URL to a [Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob) object, whether it represents a local file or a remote URL.
```

Reply:
seems like it is a copied description from `@huggingface/blob` rather than for dduf